### DISCUSSION TITLE
Stored procedures

### DISCUSSION THREAD

**Discussion by timuckun** [prest/prest#587] [API Server] [answered]
I don't see any documentation on how to call stored procs. I presume this is trivial but some documentation would be nice,

**Comment by avelino** [comment, marked as answer]
do you want to list postgres objects (functions, views, ...)?

if so the endpoint `/tables` returns this for you

**Reply by timuckun** [reply]
No I am wondering how I call stored procs with prest.  Say do I do the equivalent of 

select * from somefunc(param1 => 'xxx', param2 =>123)

**Reply by timuckun** [reply]
BTW where are these endpoints documented?

**Reply by avelino** [reply]
here is the [filter documentation](https://docs.prestd.com/query-statements/#filter-where), but it would look something like this

```
/{DATABASE}/{SCHEMA}/somefunc?param1=$eq.xxx&param2=$eq.123
```

`somefunc` (stored procs) is in place of the table

**Reply by timuckun** [reply]
Oh OK great. If I can treat it like a table that makes it nice.

I'll try it with some different return types and let you know.

**Comment by timuckun** [comment]
Thanks, I'll try it with various return types and let you know...

BTW I am getting a "database not found" error even though /tables returns the tables.

What's going on there?

**Reply by avelino** [reply]
reply on issue https://github.com/prest/prest/issues/590#issuecomment-910826973
